pkg/controller/direct/run/mapper.validation.txt

```diff
@@ -0,0 +1 @@
+[mismatch_ref] Kind=EnvVarSource, Version=v1beta1, KRM field "SecretKeyRef" has no matching proto field found (make sure the KRM field is the pascal case of the proto with "Ref" suffix)
```

Comment:
This is exactly the case we want to catch. Though the RunJob is a TF-beta migration.